resources/views/includes/elements/payment-options.blade.php

```diff
@@ -1,59 +1,92 @@
+@if (isset($variant) && $variant === 'the-marlay-group')
 <div class="container mt-4 p-5">
     <div class="card mt-4 col-md-12">
+        <div class="row mb-4 no-gutters">
+            <h3 class="lead p-heading p-heading--single">Become a Marlay Group Patron</h3>
+            <hr />
+            <div class="col-md-6 border d-flex rounded text-center p-container p-container--single">
+                <figure class="fluid">
+                    <img class="img-fluid p-image p-image--single" src="/images/payment-options/the-marlay-group.jpg"
+                        alt="person in museum" />
+                </figure>
+            </div>
+            <div class="col-md-6 d-flex flex-column justify-content-center">
+                <a href="https://fitz-tnew-test.tnhs.cloud/donate/contribute1?ct=1"
```

Comment:
This URL will be OK while we're testing it on staging, but we will need to change this to use the live URL on the production environment.

My recommendation would be to set a new environment variable (e.g. `tessitura_booking_domain`) that we can set in the `.env` file. That way it will automatically change on staging and live.

The new template code would then be:

```<a href="{{ tessitura_booking_domain }}/donate/contribute1?ct=1"```

Please ask either myself or Gareth for help changing the values in the `.env` file

Reply:
Done. 👍🏼